components/Frame/Popover/Nav.js

```diff
@@ -144,6 +144,14 @@ const Nav = ({
                   marginBottom: inIOS && !inNativeApp ? 64 : 24
                 }}
               >
+                <NavLink
+                  large
+                  href='/vote/2021'
+                  active={active}
+                  closeHandler={closeHandler}
+                >
+                  Wählen
```

Comment:
```suggestion
                  Urabstimmung
```

> Die Urabstimmung bezeichnet eine Abstimmung, zu der alle Mitglieder einer Organisation (z. B. politische Parteien, Gewerkschaften, Vereine, Studierendenschaften) aufgerufen sind.

AFAIK that's also what we had in the past and works for votes and elections.